bazel/base_images.bzl

```diff
@@ -0,0 +1,27 @@
+load("@io_bazel_rules_docker//repositories:go_repositories.bzl", _go_deps = "go_deps")
+load("@rules_oci//oci:pull.bzl", "oci_pull")
+
+def go_base_images():
+    # This call is technically optional but is included here for safety as it is idempotent and required,
+    _go_deps()
+
+    oci_pull(
```

Comment:
could you add a comment on how you generated the sha256: values and how one would update them?

https://github.com/GoogleContainerTools/distroless is not doing releases.